src/payload-transformer.spec.ts

```diff
@@ -51,14 +52,14 @@ describe('PayloadTransformer', () => {
         });
 
         it('transforms error into a response object', () => {
-            const xhr = {
+            const xhr: XMLHttpRequest = {
```

Comment:
Maybe just leave it as `any` if you can't get the typing to work without a lot of rework.